contexts/MembershipContext/src/Domain/Model/Application.php

```diff
@@ -103,4 +113,46 @@ public function needsModeration(): bool {
 		return $this->needsModeration === self::NEEDS_MODERATION;
 	}
 
+	public function isConfirmed(): bool {
+		return $this->isConfirmed === self::IS_CONFIRMED;
+	}
+
+	public function setConfirmed() {
```

Comment:
maybe call it `confirm` or `confirmMembership` instead, similar to `confirmBooked` in the `Donation` Domain model?